makefiles/lsof.mk

```
@@ -5,12 +5,14 @@ endif
 ifeq (,$(findstring darwin,$(MEMO_TARGET)))
 
 SUBPROJECTS  += lsof
-LSOF_VERSION := 62
+LSOF_VERSION := 70
 DEB_LSOF_V   ?= $(LSOF_VERSION)
 
 lsof-setup: setup
-	wget -q -nc -P $(BUILD_SOURCE) https://opensource.apple.com/tarballs/lsof/lsof-$(LSOF_VERSION).tar.gz
-	$(call EXTRACT_TAR,lsof-$(LSOF_VERSION).tar.gz,lsof-$(LSOF_VERSION),lsof)
+	$(call GITHUB_ARCHIVE,apple-oss-distributions,lsof,$(LSOF_VERSION),lsof-$(LSOF_VERSION))
+	$(call EXTRACT_TAR,lsof-$(LSOF_VERSION).tar.gz,lsof-lsof-$(LSOF_VERSION),lsof)
+	$(call DO_PATCH,lsof,lsof,-p1)
+	
```

Delete this blank line

```suggestion
	$(call DO_PATCH,lsof,lsof,-p1)
```